src/bin/tester.rs

```diff
@@ -163,92 +247,85 @@ fn main() {
             });
         }
         Command::Dir { verbose, dir } => {
-            // Counters for reporting overall stats at the end.
-            let mut total_count = 0;
-            let mut unambiguous_count = 0;
-            let mut ambiguous_count = 0;
-            let mut too_short_count = 0;
-            let mut no_parse_count = 0;
+            // HACK(eddyb) avoid parsing some files that hit
+            // `lykenware/gll` worst-cases (many GBs of RAM usage)
+            // FIXME(eddyb) fix the problems (e.g. implement GC).
+            const BLACKLIST: &[&str] = &[
+                "libcore/unicode/tables.rs",
+                "issues/issue-29466.rs",
+                "issues/issue-29227.rs",
+            ];
 
             // Find all the `.rs` files inside the desired directory.
             let files = WalkDir::new(dir)
                 .contents_first(true)
                 .into_iter()
                 .map(|entry| entry.unwrap())
-                .filter(|entry| entry.path().extension().map_or(false, |ext| ext == "rs"));
-
-            let mut stdout = io::stdout();
+                .filter(|entry| entry.path().extension().map_or(false, |ext| ext == "rs"))
+                .filter(|entry| !BLACKLIST.iter().any(|&b| entry.path().ends_with(b)));
```

Comment:
I'd extract out to a function `is_blacklisted` so we can iterate on the blacklisting mechanism there.